lib/apiconfig/apiconfig.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2017,2020 Tigera, Inc. All rights reserved.
```

Comment:
Do we have detailed guidance on this?  Personally I would have changed it to `2016-2020`.

Reply:
The pre-commit check, which prompted me to update this, showed a comma separated list was a valid format. I didn't look any deeper into it.

Reply:
Here is what is in the [pre-commit hook ](https://github.com/projectcalico/libcalico-go/blob/master/git-hooks/pre-commit#L67)
```
  echo "Example for updated files (use commas and year ranges):"
  echo "  Copyright (c) 2012,2015-${year} ${copyright_owner}. All rights reserved."
```